core/src/js/components/mercenaries/overlay/teams/mercenaries-out-of-combat-player-team.component.ts

```diff
@@ -106,8 +106,9 @@ export class MercenariesOutOfCombatPlayerTeamComponent
 										const refEquipment = refMerc.equipments?.find(
 											(e) => e.equipmentId === equip.Id,
 										);
+										const equipCardId = this.allCards.getCardFromDbfId(refEquipment?.cardDbfId)?.id.replace(/[1-4]$/, equip.Tier.toString());
 										return BattleEquipment.create({
-											cardId: this.allCards.getCardFromDbfId(refEquipment?.cardDbfId)?.id,
+											cardId: equipCardId,
```

Comment:
Good catch.

I would however try to avoid using `replace` if possible, because it is brittle.
How about something like
```
const refEquipment = refMerc.equipments?.find(
    (e) => e.equipmentId === equip.Id,
);
const refTier = refEquipment.tiers.find((t) => t.tier === equip.Tier);
return BattleEquipment.create({
    cardId: this.allCards.getCard(refTier?.cardDbfId)?.id,
    level: equip.Tier,
});
```

Reply:
Done